meson.build

```diff
@@ -181,6 +183,7 @@ endif
 # Optional dependency on GObject
 build_gobject = false
 if get_option('enable-gobject-types')
+  graphene_gobject_string = '@0@-gobject-@1@'.format(meson.project_name(), graphene_api_version)
```

Comment:
If we use `graphene_api_path` above, then this should be changed into `graphene_gobject_api_path`.